czech/kapsy/nuz/bandaz/marsmelak.md~

```diff
@@ -0,0 +1,6 @@
+Tak je to jasné, je to zatraceně dobrá simulace, protože kdyby to byla realita, 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.